src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -357,5 +379,28 @@ public static IQueryable<Publication> HydratePublicationForPublicationViewModel(
                 .ThenInclude(p => p.Methodology)
                 .ThenInclude(p => p.Versions);
         }
+
+        private List<MyPublicationViewModel> HydrateMyPublicationsViewModel(List<Publication> publications)
```

Comment:
Given it's returning a list, I'd call this `HydrateMyPublicationsViewModels`

Reply:
And given that it;s also doing ordering behaviour, I'd probably either mention that in the method title or have whatever needs the ordering to do the ordering itself after calling this.

So maybe either rename to `HydrateAndOrderMyPublicationsViewModels`, or remove the ordering and have the caller do it.

Not a biggie tbh, but showing your intent / side effects with the method names is a good habit to get into :)

Reply:
Renamed - Did consider moving the ordering out of the method for the reasons you mentioned.